src/test/unit/flight_failsafe_unittest.cc

```diff
@@ -39,6 +43,9 @@ extern "C" {
     #include "flight/failsafe.h"
 
     failsafeState_t* failsafeInit(rxConfig_t *intialRxConfig);
+
+    rcControlsConfig_t testRcControlsConfig[MAX_PROFILE_COUNT];
```

Comment:
It§s not that much about typing ... but if internals change, its much easier to modify few macros. Also it is much easier to find affected parts of code when macros is used.